src/webots/gui/WbConsole.cpp

```diff
@@ -272,29 +272,36 @@ QString WbConsole::htmlSpan(const QString &s, WbLog::Level level) const {
   if (s.isEmpty() || s == "\n")
     return "";
 
-  QString color;
+  QString fgColor;
```

Comment:
https://github.com/cyberbotics/webots/wiki/Coding-Style#cs44-no-abbreviations
```suggestion
  QString foregroundColor;
```